templates/website/pol-info-survey.html

```diff
@@ -0,0 +1,21 @@
+<?php	
+	
+$values['title'] = "Help us by answering a few quick questions";	
+$values['robots'] = 'noindex, nofollow';	
+	
+template_draw('header', $values);	
+	
+?>
+
+<script type="text/javascript" >
+document.write('<script src="http' + ( ("https:" == document.location.protocol) ? "s" : "") + 
+                '://www.surveygizmo.com/s3/4342879/WTT-Political-Information-Survey?__no_style=true&__output=embedjs&__ref=' +
+                escape(document.location.origin + document.location.pathname) +
```

Comment:
This would pass the survey unique token in, which links. Presumably ref is not required, they're all coming from the same place?

Reply:
Ahh, it uses the URL as a ref. Yep I can adjust that to be consistent.